dash_bio/NeedlePlot.py

```diff
@@ -5,30 +5,32 @@
 
 class NeedlePlot(Component):
     """A NeedlePlot component.
-
+The Needle Plot component is used to display large datasets
+containing categorical or numerical data. The lines and markers in
+the plot correspond to bars in a histogram.
 
 Keyword arguments:
 - id (string; optional): The ID of this component, used to identify dash components
 in callbacks. The ID needs to be unique across all of the
 components in an app.
-- mutationData (optional): . mutationData has the following type: dict containing keys 'x', 'y', 'mutationGroups', 'domains'.
+- mutationData (optional): The data that are displayed on the plot.. mutationData has the following type: dict containing keys 'x', 'y', 'mutationGroups', 'domains'.
 Those keys have the following types:
   - x (string | list; optional)
   - y (string | list; optional)
   - mutationGroups (list; optional)
   - domains (list; optional)
-- xlabel (string; optional)
-- ylabel (string; optional)
-- rangeSlider (boolean; optional)
-- needleStyle (optional): . needleStyle has the following type: dict containing keys 'stemColor', 'stemThickness', 'stemConstHeight', 'headSize', 'headColor', 'headSymbol'.
+- xlabel (string; optional): Title of the x-axis.
+- ylabel (string; optional): Title of the y-axis.
+- rangeSlider (boolean; optional): If True, enables a rangeslider for the x-axis.
```

Comment:
Whenever describing a boolean in docstrings, it would help to use sentence case, I think. When I see `True` instead of `true` I immediately think Python, and these are the same docstrings that will get inserted into R (and any other language) documentation too 🤓